templates/disable_inactive_users.j2.sh

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+###
+# We're running as root, so login to kerberos using the host's keytab
+#
+# It would be better to create a kerberos service instead, but
+# that is left as a future improvement.
+###
+kinit -k -t /etc/krb5.keytab
+
+###
+# Extract the domain from the host keytab
+#
+# This is some potent sed fu.  Match the first line containing
+# "host/something1@something2", then convert something2 to lower case
+# and print it, and finally quit sed.
+###
+domain=$(klist -k /etc/krb5.keytab \
+             | sed --quiet "/.*host\/\(.*\)@\(.*\)/{s//\L\2/p;q}")
```

Comment:
Nooice. I need some of this sauce over in the OpenVPN script: 
https://github.com/cisagov/ansible-role-openvpn/blob/develop/files/verify-cn.py

TASTY!  🍝